### DISCUSSION TITLE
Organizations only have up to 10 collections

### DISCUSSION THREAD

**Discussion by pascal-pws** [dani-garcia/vaultwarden#4253] [Q&A]
I have a fresh installation of Vaultwarden where I get the message that I can only create ten collections in a free organization.

![vaultwarden_error_message](https://github.com/dani-garcia/vaultwarden/assets/88563417/5ecc4259-09b0-4921-8134-47eef7359432)

Because I host it myself, I thought I had no restrictions here. This is also what I read in the forum:

https://vaultwarden.discourse.group/t/how-many-users-are-allowed-in-an-organization/1613

I have installed the following versions:

![vaultwarden_versions](https://github.com/dani-garcia/vaultwarden/assets/88563417/b60898f2-e84c-4cae-9ebf-725b4e56a183)

What could be the reason that I cannot create more than ten collections?

**Comment by stefan0xC** [comment]
Because we have not caught and patched out this upstream change (we are still using `v2023.12.0` in `testing`) and/or I could have made a mistake when updating our patches for `v2024.1.0`. Nevertheless, thanks for the report. I will look into it if we can free the web-vault again, so this works without limitations.

**Reply by stefan0xC** [reply]
Mea culpa. It should be fixed by https://github.com/dani-garcia/bw_web_builds/pull/153

**Comment by pascal-pws** [comment]
First of all, thank you for fixing the problem so quickly!

In this context, I have another error that I don't quite understand. As soon as I try to delete all selected collections at once:

![delete_some_collections](https://github.com/dani-garcia/vaultwarden/assets/88563417/e4fef918-9769-4fcd-8c1a-c5536b84c222)

I get the following error message in the web interface:

![web_error_message](https://github.com/dani-garcia/vaultwarden/assets/88563417/335d8610-a70b-4a52-88a2-dbb19f92a3ee)

But if I delete a collection individually, it works without any problems:

![delete_one_collection](https://github.com/dani-garcia/vaultwarden/assets/88563417/86992353-5349-418b-8d8d-0a2127966067)

What could be the reason for this?

**Reply by BlackDex** [reply]
Please create an issue for this.

**Reply by BlackDex** [reply]
I already fixed it via https://github.com/dani-garcia/vaultwarden/pull/4257

**Reply by pascal-pws** [reply]
Thanks for fixing it.
